Source Main 5.2/source/GM_Raklion.cpp

```diff
@@ -1905,7 +1905,7 @@ bool CGM_Raklion::RenderMonsterVisual(CHARACTER* c, OBJECT* o, BMD* b)
                         OBJECT* to = &tc->Object;
                         if (10 <= c->AttackTime && to->Live)
                         {
-                            to->m_byHurtByOneToOne = 35;
+                            to->m_byHurtByDeathstab = 35;
```

Comment:
hey @sven-n, do we also need to update this initial value based on the fps animation factor? or is this fine, and we only need to update the part where it's decremented?